module/atom/ocr.py

```
@@ -49,7 +49,11 @@ def coord(self) -> tuple:
         else:
             area = self.roi
 
-        x, y, w, h = self.area
+		# 原代码：x, y, w, h = self.area
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)